packages/flutter_tools/test/src/fake_vm_services.dart

```diff
@@ -52,6 +52,7 @@ class FakeVmServiceHost {
           'id': request['id'],
           'error': <String, Object?>{
             'code': fakeRequest.errorCode,
+            'message': 'error',
```

Comment:
`message` is required to be a string in 
https://github.com/dart-lang/sdk/blob/7d3d835bc9f8b89675f55cbadba1a514aa75db81/pkg/vm_service/lib/src/vm_service.dart#L2341